src/datagrid/DataGrid.stories.tsx

```diff
@@ -233,15 +234,27 @@ storiesOf("DataGrid", module)
             />
         </div>
     ))
+    .add("Grid with HIde and Show Column", () => (
```

Comment:
Observation: On navigating to the second page and then selecting all the check boxes, when we again navigate to the first page then only the select all checkbox is selected while the other checkboxes are not selected.

Reply:
done. thanks for noticing this.